apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -2,9 +2,18 @@ import { Meta } from '@storybook/addon-docs';
 
 <Meta title="Concepts/Developer/Browser Support Matrix" />
 
-## Browser Support Matrix
+Defining a browser support matrix to account for all different consumer cases is a difficult undertaking.
+Browser matrix support takes into account various factors such as ECMAScript (ES), browser, and CSS
+feature compatibility across different browsers and across different versions of those browsers.
+FluentUI strives to find a balance that enables consumers who support modern browsers to reap the
+full benefits of the library while at the same time providing the necessary guidance to consumers who
+support older browsers. Details of a full and partial browser support matrix as well as actions consumers
+will need to take such as transpiling and polyfilling are outlined below.
```

Comment:
In terms of voice here, we should avoid "will need to take", "must take", etc. Instead, it's preferred to say "can take" "can take to mitigate x", etc.